mu/interface/dialogs.py

```diff
@@ -224,7 +225,11 @@ def setup(self, log, settings, packages):
         self.tabs.addTab(self.microbit_widget, _('BBC micro:bit Settings'))
         self.package_widget = PackagesWidget()
         self.package_widget.setup(packages)
-        self.tabs.addTab(self.package_widget, _('Third Party Packages'))
+        if 'armv' not in platform.platform():
+            # Only allow third party package management if NOT running on a
+            # Raspberry Pi. See:
+            # https://github.com/mu-editor/mu/pull/749#issuecomment-459051823
+            self.tabs.addTab(self.package_widget, _('Third Party Packages'))
```

Comment:
I think a missing tab in the settings depending on the platform could cause confusion for online material, tutorials or users trying to help other users.
I've often insisted to other people to go a specific setting in an app just to find out that the instructions I was given them were wrong because the feature was missing or somewhere else in their version they had.

Could we instead show the tab but change the content, a simple one liner saying something along of the lines of "In this platform packages can be installed in using pip3."? That way users would not wonder if they have an old version of Mu, and also will point them to the right direction.

Reply:
Right... so we're just going to go round and round in circles about this. I believe we have a good-enough solution right now, and we should get this functionality in front of users as soon as possible as an alpha release so we get *feedback* from real people. Anything else is "educated" speculation (but speculation nonetheless). 

Only with feedback can we make an properly informed decision. If we don't get feedback, we're probably OK as we are. :-)

Reply:
I would say the Plotter not being available in Raspberry Pi is a good example of missing features in different platforms causing confusion:
https://github.com/mu-editor/mu/issues/714
https://github.com/mu-editor/mu/issues/689
https://github.com/mu-editor/mu/issues/694